src/main/java/com/spikes2212/dashboard/AutoChooser.java

```diff
@@ -0,0 +1,35 @@
+package com.spikes2212.dashboard;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj2.command.Command;
+
+import java.util.HashSet;
+
+/**
+ * A class that implements the {@link SendableChooser} class in order to make an autonomous command chooser.
+ *
+ * @author Ofri Rosenbaum
+ * @see SendableChooser
+ */
+public class AutoChooser extends SendableChooser<Command> {
+
+    public AutoChooser(Command defaultOption, Command... options) {
```

Comment:
add another constructor,
it will get `Command` for the default Command, `String` for the default Command's name, and `Object... options`, the options will be parsed as following:
every even index (0,2,4,...) will be a command
every odd index (1,3,5,...) will be the command's name to be displayed.

Reply:
So, what happens if in the place the user needs to give a command, he gives a string? or when he needs to give a string, he gives a command? exception? create null instead? what happens if user gives an uneven number of objects?